packages/block-library/src/site-logo/index.php

```diff
@@ -23,7 +23,22 @@ function render_block_core_site_logo( $attributes ) {
 
 	add_filter( 'wp_get_attachment_image_src', $adjust_width_height_filter );
 	$custom_logo = get_custom_logo();
-	$classnames  = array();
+
+	if ( ! $attributes['isLink'] ) {
+		/* Remove the link */
+		$custom_logo = preg_replace( '#<a.*?>(.*?)</a>#i', '\1', $custom_logo );
+	}
+
+	if ( $attributes['isLink'] && '_blank' === $attributes['linkTarget'] ) {
+		/*
+		 * Add the link target after the rel="home".
+		 * Add an aria-label for informing that the page opens in a new tab.
+		*/
```

Comment:
We should probably use the `// style for these comments.
```suggestion
		// Add the link target after the rel="home".
		// Add an aria-label for informing that the page opens in a new tab.
```